modules/logging-release-notes-5-8-8.adoc

```diff
@@ -0,0 +1,36 @@
+// module included in /logging/logging-5-8-release-notes
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-8-8_{context}"]
+= Logging 5.8.8
+This release includes link:https://access.redhat.com/errata/RHBA-2024:3737[OpenShift Logging Bug Fix Release 5.8.8] and link:https://access.redhat.com/errata/RHBA-2024:3738[OpenShift Logging Bug Fix Release 5.8.8].
+
+[id="logging-release-notes-5-8-8-bug-fixes"]
+== Bug fixes
+
+* Before this update, there was a delay in restarting ingesters when configuring `LokiStack` as the {loki-op} set the write-ahead log `replay_memory_ceiling` to zero bytes for the `1x.demo` size. With this update, the minimum value used for the `replay_memory_ceiling` has been increased to avoid delays. (link:https://issues.redhat.com/browse/LOG-5615[LOG-5615])
```

Comment:
```suggestion
* Before this update, there was a delay in restarting Ingesters when configuring `LokiStack` as the {loki-op} set the write-ahead log `replay_memory_ceiling` to zero bytes for the `1x.demo` size. With this update, the minimum value used for the `replay_memory_ceiling` has been increased to avoid delays. (link:https://issues.redhat.com/browse/LOG-5615[LOG-5615])
```

Reply:
I did a quick search of the repo and it seemed divided on whether it should be `ingesters` or `Ingesters`.